dev-docs/bidders/adf.md

```diff
@@ -1,15 +1,17 @@
 ---
 layout: bidder
-title: AdformOpenRTB
-description: Prebid AdformOpenRTB Bidder Adaptor
-biddercode: adformOpenRTB
+title: Adf
```

Comment:
I'd suggest leaving Adform OpenRTB here -- it's more descriptive than adf.

Reply:
good point, thanks